cosmos/gaia
# v7.0.x-final

This PR updates gaia v7.0.x.  It is an omnibus. 

Closes:

* #2078 

Changes:
* use informal's tendermint fork and bump to v0.34.26
* bump sdk to v0.45.12
* bump ibc (not state breaking) to v3.0.2
* fix state sync script 
* don't use a t-word docker image registry
* markdownlint
* fix golangci-lint 
* use iavl fast node by default 

These are combined here, because this is to be a final cut of release/v7.0.x

I do wish to politely request that we begin to maintain all pieces of cosmoshub-4, beginning with v4.2.x, once we have v8 shipping

To test state sync:

bash contrib/statesync.bash

test-liveness is expected to fail until this is merged.  At that time, we can add to readme.md, the invocation:

```bash
bash contrib/statesync.bash
```

and we shall witness its passage. 

This PR should sync from the first block of v7 to the last, to ensure that it is good.
